packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -28,7 +28,7 @@ import {
 import { LabelSet } from './LabelSet';
 import { Batcher, UngroupedBatcher } from './export/Batcher';
 import { PushController } from './export/Controller';
-import { NoopExporter } from '../test/mocks/Exporter';
+import { NoopExporter } from './export/NoopExporter';
```

Comment:
Another option is to use `ConsoleMetricExporter` as a default metric exporter.

Reply:
or `InMemoryMetricExporter` similar to `InMemorySpanExporter`.

Reply:
I think no-op is better. We can create a more fancy version of no-op if a need arises later.